exercises/palindrome/src/example.h

```diff
@@ -0,0 +1,25 @@
+#ifndef PALINDROME_H
+#define PALINDROME_H
+
+/* structure containing a pair of integer */
+typedef struct paire {
+   int a;
+   int b;
+} Paire;
+
+/* structure containing a palindrome and an array of his factors */
+typedef struct palindProduct {
+   int palindrome;
+   Paire *factors;
+} PalindromeProduct;
+
+/* Return 0 if the product of a and b is a palindrome, 1 otherwise */
+int isPalindrome(int a, int b);
```

Comment:
I wanted to test every function i used,, but it might be too much. 
`
void test_palindrome_even(void)
{
   TEST_ASSERT_EQUAL_INT(0, isPalindrome(91, 99));
}`
I'll remove these tests